AutoPr-ComputerVision-toothache-REST Spec PrNumber 5721

adxsdknet opened the pull request.

adxsdknet committed:
.NET SDK Resource Provider:'ComputerVision'
REST Spec PR 'https://github.com/Azure/azure-rest-api-specs/pull/5721'
REST Spec PR Author 'toothache'
REST Spec PR Last commit

toothache commented:
The latest change in Ocr swagger removes `mode` parameter from `BatchReadFile` operation, and thus is a breaking change. We need to modify corresponding tests file.

adxsdknet committed:
.NET SDK Resource Provider:'ComputerVision'
REST Spec PR 'https://github.com/Azure/azure-rest-api-specs/pull/5721'
REST Spec PR Author 'toothache'
REST Spec PR Last commit

toothache commented:
I fixed broken tests via [303e86d](https://github.com/toothache/azure-sdk-for-net/commit/303e86d057542cc5f78e67c95f3fd2ef68f9521c). Should I create a manual PR for this change?

adxsdknet committed:
.NET SDK Resource Provider:'ComputerVision'
REST Spec PR 'https://github.com/Azure/azure-rest-api-specs/pull/5721'
REST Spec PR Author 'toothache'
REST Spec PR Last commit

dsgouda reviewed (approved):
LGTM
